python/nav/web/templates/navlets/status2_edit.html

```diff
@@ -8,7 +7,11 @@
       Refresh interval (seconds)
       <input type="text" name="interval" value="{{ interval }}">
     </label>
-    {% crispy form %}
+    {% if form.attrs %}
+      {% include 'custom_crispy_templates/_form_content.html' %}
+    {% else %}
+      {{ form }}
+    {% endif %}
```

Comment:
I believe we stopped with the `{% if form.attrs %}` stuff ?

Reply:
Yes, we did, this is just an old PR, will fix that!

Reply:
Just for documentation's sake: We discussed this in #3140 and #3157